lib/scan/dir/engine.py

```diff
@@ -251,3 +240,5 @@ def start(target, users, passwds, ports, timeout_sec, thread_number, num, total,
         os.remove(thread_tmp_filename)
     else:
         warn(messages(language, "input_target_error").format('dir_scan', target))
+        
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing whitespace](https://app.codacy.com/manual/zdresearch/OWASP-Nettacker/pullRequest?prid=5065631)

Reply:
I have updated the code